src/sentry/static/sentry/app/bootstrap.jsx

```diff
@@ -44,6 +43,52 @@ Sentry.configureScope(scope => {
   if (window.__SENTRY__VERSION) {
     scope.setTag('sentry_version', window.__SENTRY__VERSION);
   }
+  // TODO(daniel): Maybe we need to follows from
+  // This is the inital pageload span
+  scope.setSpan(
+    Sentry.getCurrentHub().startSpan({
+      op: 'pageload',
+      sampled: true,
+    })
+  );
+});
+
+// APM --------------------------------------------------------------
+let flushTransactionTimeout = undefined;
+let firstPageLoad = true;
+
+Router.browserHistory.listen(location => {
+  if (location && location.action === 'REPLACE') {
+    // We do set the transaction name in the router but we want to start it here
+    // since in the App component where we set the transaction name, it's called multiple
+    // times. This would result in loosing the start of the transaction.
+    let transactionSpan;
+    Sentry.getCurrentHub().configureScope(scope => {
```

Comment:
Same thought here. Take the return value of `Sentry.getCurrentHub()` and use throughout the function.